bench_test.go

```diff
@@ -0,0 +1,652 @@
+	{"POST", "/repos/:owner/:repo/hooks/:id/tests"},
+	{"DELETE", "/repos/:owner/:repo/hooks/:id"},
+	{"POST", "/repos/:owner/:repo/merges"},
+	{"GET", "/repos/:owner/:repo/releases"},
+	{"GET", "/repos/:owner/:repo/releases/:id"},
+	{"POST", "/repos/:owner/:repo/releases"},
+	//{"PATCH", "/repos/:owner/:repo/releases/:id"},
+	{"DELETE", "/repos/:owner/:repo/releases/:id"},
+	{"GET", "/repos/:owner/:repo/releases/:id/assets"},
+	{"GET", "/repos/:owner/:repo/stats/contributors"},
+	{"GET", "/repos/:owner/:repo/stats/commit_activity"},
+	{"GET", "/repos/:owner/:repo/stats/code_frequency"},
+	{"GET", "/repos/:owner/:repo/stats/participation"},
+	{"GET", "/repos/:owner/:repo/stats/punch_card"},
+	{"GET", "/repos/:owner/:repo/statuses/:ref"},
+	{"POST", "/repos/:owner/:repo/statuses/:ref"},
+
+	// Search
+	{"GET", "/search/repositories"},
+	{"GET", "/search/code"},
+	{"GET", "/search/issues"},
+	{"GET", "/search/users"},
+	{"GET", "/legacy/issues/search/:owner/:repository/:state/:keyword"},
+	{"GET", "/legacy/repos/search/:keyword"},
+	{"GET", "/legacy/user/search/:keyword"},
+	{"GET", "/legacy/user/email/:email"},
+
+	// Users
+	{"GET", "/users/:user"},
+	{"GET", "/user"},
+	//{"PATCH", "/user"},
+	{"GET", "/users"},
+	{"GET", "/user/emails"},
+	{"POST", "/user/emails"},
+	{"DELETE", "/user/emails"},
+	{"GET", "/users/:user/followers"},
+	{"GET", "/user/followers"},
+	{"GET", "/users/:user/following"},
+	{"GET", "/user/following"},
+	{"GET", "/user/following/:user"},
+	{"GET", "/users/:user/following/:target_user"},
+	{"PUT", "/user/following/:user"},
+	{"DELETE", "/user/following/:user"},
+	{"GET", "/users/:user/keys"},
+	{"GET", "/user/keys"},
+	{"GET", "/user/keys/:id"},
+	{"POST", "/user/keys"},
+	//{"PATCH", "/user/keys/:id"},
+	{"DELETE", "/user/keys/:id"},
+}
+
+var gplusAPIRoutes = []route{
+	// People
+	{"GET", "/people/:userId"},
+	{"GET", "/people"},
+	{"GET", "/activities/:activityId/people/:collection"},
+	{"GET", "/people/:userId/people/:collection"},
+	{"GET", "/people/:userId/openIdConnect"},
+
+	// Activities
+	{"GET", "/people/:userId/activities/:collection"},
+	{"GET", "/activities/:activityId"},
+	{"GET", "/activities"},
+
+	// Comments
+	{"GET", "/activities/:activityId/comments"},
+	{"GET", "/comments/:commentId"},
+
+	// Moments
+	{"POST", "/people/:userId/moments/:collection"},
+	{"GET", "/people/:userId/moments/:collection"},
+	{"DELETE", "/moments/:id"},
+}
+
+func calcMem(name string, load func()) {
+	m := new(runtime.MemStats)
+
+	// before
+	runtime.GC()
+	runtime.GC()
+	runtime.GC()
+	runtime.GC()
+	runtime.ReadMemStats(m)
+	before := m.HeapAlloc
+
+	load()
+
+	// after
+	runtime.GC()
+	runtime.GC()
+	runtime.GC()
+	runtime.GC()
+	runtime.ReadMemStats(m)
+	after := m.HeapAlloc
+
+	println(""+name+":", after-before, "Bytes")
+}
+
+func goyaveHandler(res *Response, req *Request) {
+	res.String(http.StatusOK, "OK")
+}
+
+func loadGoyaveRouter(routes []route) *Router {
+	config.Load()
```

Comment:
```suggestion
```
If this benchmark is focused on performance once the server is started, config doesn't need to be loaded. Config and language files are only loaded when `goyave.Start()` is called.